db/migrate/20200717065508_create_clients.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateClients < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
I'm not 100% sure but maybe it makes sense to set `null: false` for the `name`, `public_key` and `api_key` as a client without any of these fields might not be valid. Also, regarding `unique: true`, I would add an index to these fields. 😊